admin_ipsandports.php

```diff
@@ -137,7 +140,10 @@
 					standard_error('cantdeletedefaultip');
 				}
 			} else {
-				standard_error('ipstillhasdomains');
+				if($result_checkdomain['id'] != '')
```

Comment:
please use curly braces to enclose the if-statements (like everywhere in the code)

Reply:
fixed